app/app_controller.rb

```diff
@@ -0,0 +1,3 @@
+class MakersBnB < Sinatra::Base
+  set :public_folder, Proc.new { File.join(root, 'static') }
```

Comment:
What is the reason for having app_controller file? Why can this not be inside of app.rb? Can we try and combine this together?